cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -27,27 +27,28 @@
 include(FindPackageHandleStandardArgs)
 include(SelectLibraryConfigurations)
 
-find_path(ZFP_INCLUDE_DIR
+find_path(ZFP_INCLUDE_DIRS
           NAMES zfp.h
-          PATHS $ENV{ZFP_ROOT}/inc
+          PATHS $ENV{ZFP_DIR}/include
```

Comment:
Well, they changed build system (so header/library position in the filesystem), folder names and also API (so changes in the code are required).
They do not provide versioning, so even if we find all the pieces, we shall have some ifdef in the code depending on the version, but I don't see an easy way to detect the version of the library.

Since this is gonna be the actual first release of this plugin, I believe that starting with a clean situation is better. CMake will tell the required version of the library is 0.5.1, so it is well clear to the user where is the problem.
In case the user has an older version he/she just need to update it.